tarantool/connection.py

```diff
@@ -796,6 +804,14 @@ def _send_request_wo_reconnect(self, request):
                 self.update_schema(e.schema_version)
                 continue
 
+        while response._code == IPROTO_CHUNK:
+            if on_push is not None: 
+                if callable(on_push):
```

Comment:
It is better to check it once in the object constructor.

Reply:
Done, moved `callable` check to methods `call`, `eval`, `insert`, ...

Reply:
Thank you. I like it. @DifferentialOrange , could you offer a better solution?
Anyway, @GRISHNOV , be free to resolve the conversation.

Reply:
I think `callable` is fine (and I'm not sure that there is any better approach).

In common, I thing it would be nice to use modern Python [type hints](https://docs.python.org/3/library/typing.html) some day because for now ~90% of arguments do not have any type asserts at all.